cassandra-scheduler/src/main/java/com/mesosphere/dcos/cassandra/scheduler/resources/TasksResource.java

```diff
@@ -141,8 +145,9 @@ public void replace(@QueryParam("node") final String name)
     @GET
     @Path("connect")
```

Comment:
I think this interface is left here for compatibility reasons, the calls were all marked @Deprecated and just call into ConnectionResource, which has the `/connection` endpoint.